Tensile/TensileCreateLibrary.py

```diff
@@ -1183,7 +1187,7 @@ def splitExtraParameters(par):
   arguments["Architecture"] = args.Architecture
   arguments["SeparateArchitectures"] = args.SeparateArchitectures
   arguments["LazyLibraryLoading"] = args.LazyLibraryLoading
-  arguments["CxxCompiler"] = args.CxxCompiler
+  arguments["CxxCompiler"] = CxxCompiler
```

Comment:
You corrected args so not sure why you would not trust it here but use it later,  maintenance problem